fmn/rules/fedbadges.py

```diff
@@ -25,7 +25,7 @@ def fedbadges_person_first_login(config, message):
 
 @hint(topics=[_('fedbadges.person.rank.advance')])
 def fedbadges_person_rank_advance(config, message):
-    """ Badges: The rank of someone changed on the badges leaderboard
+    """ Fedora Badges rank changes
```

Comment:
Here we specify 'Fedora' for the fist time, do we want it? Should we add it for new badges? (New Fedora badge awarded?)